source/js/components/search-form.js

```diff
@@ -0,0 +1,108 @@
+import fetchUtils from "@degjs/fetch-utils";
+
+const searchForm = () => {
+
+
+    const formEl = document.querySelector('.js-search-form');
+    const layoutEl = document.querySelector('.js-general-content');
+    const searchFieldSet = document.querySelector('.js-search-fieldset');
```

Comment:
[suggestion] Not the end of the world, but you already queried `searchInputEl`, so this is just an additional DOM query that didn't need to happen (you could've just disabled the input rather than the fieldset).